compose/elasticsearch.yml

```diff
@@ -1,7 +1,7 @@
 version: '2.4'
 services:
   ccd-elasticsearch:
-    image: docker.elastic.co/elasticsearch/elasticsearch:6.4.2
+    image: docker.elastic.co/elasticsearch/elasticsearch:6.8.14
```

Comment:
I don't think we should actually be updating the ccd-docker version of ES? Shouldn't we keep it as the same version in production?